projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.js

```diff
@@ -109,6 +109,40 @@
 				}
 			);
 		}
+
+		if ( document.querySelector('body.jetpack-masterbar') ) {
+			document.querySelectorAll( 'li.wp-has-submenu.wp-not-current-submenu' ).forEach( function ( el ) {
+				const submenu = el.querySelector( '.wp-submenu' );
+				const linkElement = el.querySelector( 'a' );
+
+				el.addEventListener( 'mouseover', function() {
+					submenu.style.display = null;
+					submenu.style.top = '-1px';
+					if ( ! isElementInViewport( submenu ) ) {
+						submenu.style.top = ( 34 - submenu.clientHeight ) + 'px';
```

Comment:
Just out of curiosity - what's the reasoning for choosing 34px? Why does it have to be so specific?

Reply:
34px is the height of the menu item, I will add a comment to make it more clear.

Reply:
Thanks for the disclaimer. Wouldn't it be more appropriate to calculate the height of the menu item, rather than hardcoding it?